fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/DateLiteral.java

```diff
@@ -189,19 +189,23 @@ static Result<String, AnalysisException> normalize(String s) {
                 if (len == 4 || len == 2) {
                     sb.append(s, i, j);
                 } else if (len == 3) {
-                    if (partNumber == 0) {
-                        String yy = s.substring(i, i + 2);
-                        int year = Integer.parseInt(yy);
-                        if (year >= 0 && year <= 69) {
-                            sb.append("20");
-                        } else if (year >= 70 && year <= 99) {
-                            sb.append("19");
+                    if (s.charAt(j) == '.') {
```

Comment:
why "201.01.01" is year 2020, but "201-01-01" is year 201

Reply:
this behavior is align to mysql